### DISCUSSION TITLE
Simple one: how to put <model-viewer> and an <mg ...> side by side

### DISCUSSION THREAD

**Discussion by motionmountain** [google/model-viewer#4144] [Q&A] [answered]
In my webpage, the < model-viewer ...> < /model-viewer > always takes a complete line. 
It behaves like having a < br > built-in.

What can be done to have
< model-viewer ... >< /model-viewer > and < img ... >
displayed side by side on a html page?

**Comment by elalish** [comment, marked as answer]
We're styled internally as a `display: block` div. All the usual CSS layout stuff to should apply to MV the same as to a `<div>`.
